commands_to_load/Mod.py

```diff
@@ -82,5 +83,398 @@ def __init__(self, bot):
+        if eObj is not False:
+            await council.send(embed=eObj)
+
+    @commands.command()
+    async def clear(self, ctx, numOfMsgs = 10.0):
+        # Deletes the last X (10 by default) msg's from channel
+        # Limits: Max 100 messages at a time AND no messages older than 2 weeks (14 days)
+
+        logger.info('[Mod clear()] clear function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod clear()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod clear()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        numOfMsgs = int(numOfMsgs)
+
+        # Verify args
+        if numOfMsgs > 100 or numOfMsgs < 1: 
+            # Prevents discord.ClientException 
+            eObj = await em(ctx, description='Number of messages to be between 1 and 100 inclusively', footer='Invalid arguments')
+            if eObj is not False:
+                await ctx.send(embed=eObj)
+
+        channel = ctx.channel
+        # Grab the last X messages from the channel regardless of user
+        logger.info('[Mod clear()] grabbing last {} message from {}'.format(numOfMsgs, channel))
+        messages = await channel.history(limit=numOfMsgs).flatten()
+        logger.info('[Mod clear()] messages to be deleted: {}'.format(messages))
+        
+        try:
+            await channel.delete_messages(messages)
+            logger.info('[Mod clear()] messages from {} deleted'.format(channel))
+        except discord.HTTPException:
+            eObj = await em(ctx, description='Messages cannot be older than 2 weeks', footer='Command Error')
+            if eObj is not False:
+                await ctx.send(embed=eObj, delete_after=10.0)
+            return
+
+        eObj = await em(ctx, description='{} messages deleted'.format(numOfMsgs), footer='Message will self destruct in 5 ...')
+        if eObj is not False:
+            await ctx.send(embed=eObj, delete_after=5.0)
+
+    @commands.command()
+    async def purge(self, ctx, *args):
+        # Deletes 10 message in channel from user
+        # Order of arguments doesn't matter, the code works around it
+        args = list(args)
+
+        # Verify Minion
+        logger.info('[Mod purge()] purge function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod purge()] invoking command deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod purge()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        # Verify arguments
+        logger.info('[Mod purge()] verifying arguments')
+        mentions = ctx.message.mentions
+        if len(mentions) != 1: 

```

Comment:
this should be changed into 2 separate conditional branches, one where it deals with no @s and one with it deals with too many users @ed